test/fixtures/output/environment/1.golden

```diff
@@ -1,6 +1,6 @@
-  Current |      ID      |   Name    
-----------+--------------+-----------
-          | env-123      | env123    
-          | env-595      | other     
-  *       | env-596      | default   
-          | env-srUpdate | srUpdate  
+  Current |      ID      |   Name   | Stream Governance Package  
+----------+--------------+----------+----------------------------
+          | env-123      | env123   | ESSENTIALS                 
```

Comment:
Can this value ever be "", or will it always be one of essentials or advanced?

Reply:
(If it can be "", we might want to consider tagging this field with `omitempty`)

Reply:
It should always have a value of either ESSENTIALS or ADVANCED.